devsetup/roles/download_tools/defaults/main.yaml

```diff
@@ -6,12 +6,11 @@ kuttl_version: 0.15.0
 opm_version: latest
 
 # operator-sdk version to use (must be specific version)
-#sdk_version: v0.19.2 - cnosp is right now based on that version
+# sdk_version: v0.19.2 - cnosp is right now based on that version
```

Comment:
I'm not completely sure I understand the comment here, but that isn't relevant to the subject of review.

Reply:
> I'm still not convinced about splitting variables in new lines: i mean, we can agree to change defaults `line-length` yamllint rule to 120 chars. Apart from that, the rest of the PR is ok for me. Let's wait for other comments.

I tried different style of new line splitting. The only one that works is the "double quoted" as correctly used by @amolkahat so the patch for me is ok.